content/community/team/alex.md

```diff
@@ -0,0 +1,12 @@
+---
+author: true
+alias: zen-dog
+name: Alex
+avatar: alex.jpg
+about:
+  Alex is simply awesome, but someone has yet to put his awesomeness into a brief and meaningful summary.
```

Comment:
Alex is an engineer and he solves problems. Sometimes this involves writing code.